packages/devextreme-react/gulpfile.js

```diff
@@ -65,15 +70,55 @@ gulp.task(NPM_README, gulp.series(
     () => gulp.src(config.npm.readme).pipe(gulp.dest(config.npm.dist))
 ));
 
-gulp.task(NPM_BUILD, gulp.series(
-    NPM_CLEAN,
-    gulp.parallel(NPM_LICENSE, NPM_PACKAGE, NPM_README),
+gulp.task(NPM_BUILD_ESM, gulp.series(
+    () => gulp.src([
+        config.src,
+        `!${config.testSrc}`
+    ])
+        .pipe(ts('tsconfig.esm.json'))
+        .pipe(gulp.dest(config.npm.dist + '/esm'))
+));
+
+gulp.task(NPM_BUILD_CJS, gulp.series(
     () => gulp.src([
         config.src,
         `!${config.testSrc}`
     ])
         .pipe(ts('tsconfig.json'))
-        .pipe(gulp.dest(config.npm.dist))
+        .pipe(gulp.dest(config.npm.dist + '/cjs'))
+));
+
+gulp.task(NPM_PREPARE_FOLDERS, (done) => {
+    makeModuleForFolder('common');
```

Comment:
Seems that we can create const array with folders and then use forEach to call makeModuleForFolder for them